src/platform/apollolake/include/platform/platform.h

```diff
@@ -33,25 +33,14 @@
 #ifndef __PLATFORM_PLATFORM_H__
 #define __PLATFORM_PLATFORM_H__
 
+#include <cavs/memory.h>
+
 #define PLATFORM_RESET_MHE_AT_BOOT		1
 
 #define PLATFORM_DISABLE_L2CACHE_AT_BOOT	1
 
-#define PLATFORM_CORE_COUNT			2
```

Comment:
Would be good, for the sake of config verification purpose, to keep `MAX_CORE_COUNT` h/c in the platform header (as specific platform max capability) and verify against the `CORE_COUNT` defined by user in the Kconfig during compilation (throw pre-proc #error if invalid).